libs/platform/src/lib/components/approval-flow/approval-flow.component.html

```diff
@@ -17,15 +17,13 @@ <h3 class="approval-flow__title" [attr.dir]="_dir">{{ title }}</h3>
 
 <div class="approval-flow__container">
     <div class="approval-flow__carousel-controls" *ngIf="_isCarousel" [attr.dir]="_dir">
-        <button class="approval-flow__control--prev-slide"
-                *ngIf="_carouselStep > 0"
-                (click)="previousSlide()">
+        <button class="approval-flow__control--prev-slide" *ngIf="_carouselStep > 0" (click)="previousSlide()"
+            aria-label="Go to previous slide">
             <fd-icon [glyph]="'navigation-' + (_isRTL ? 'right' : 'left') + '-arrow'"></fd-icon>
             {{ _carouselStep }}
         </button>
-        <button class="approval-flow__control--next-slide"
-                *ngIf="(_maxCarouselStep - _carouselStep) > 0"
-                (click)="nextSlide()">
+        <button class="approval-flow__control--next-slide" *ngIf="(_maxCarouselStep - _carouselStep) > 0"
+            (click)="nextSlide()" aria-label="Go to next slide">
```

Comment:
Most probably this `aria-label` should have `i18n`

Reply:
added i18n-aria-label